spotbugsTestCases/src/java/ghIssues/issue574/Issue574RegisterExtension.java

```diff
@@ -0,0 +1,16 @@
+package ghIssues;
+
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+/**
+ * Tests that "URF_UNREAD_FIELD" warning is suppressed for fields with @RegisterExtension annotation.
+ * @see <a href="https://github.com/spotbugs/spotbugs/issues/574">GitHub issue</a>
+ */
+
+
+public final class Issue574RegisterExtension {
```

Comment:
Couldn't find any tests, which calls the `performAnalysis()` function for this class.

Reply:
Isn't it analyzed in `Issue574Test.testRegisterExtensionAnnotation`?

Reply:
It's the dependency of the `Issue574RegisterExtension` class, which is analyzed there, but it itself is not analyzed, the dependency is missing, it is even listed as a missing class on the std err (or to be more precise the `BypassFileNotFoundExceptionExtension` is listed, but if you add that, the `Issue574RegisterExtension` will be missed. The dependencies outside the `spotbugsTestCases` directory will be handled by https://github.com/spotbugs/spotbugs/pull/2840, but the dependencies inside it (such as parents, nested classes), needs to be added.

Reply:
Yes, you are right, left the comment at the wrong place. I'm sorry. Can you please add `BypassFileNotFoundExceptionExtension` to the relevant test?